src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Avx.PlatformNotSupported.cs

```diff
@@ -919,16 +919,6 @@ public abstract class Avx : Sse42
         /// </summary>
         public static Vector256<T> SetHighLow<T>(Vector128<T> hi, Vector128<T> lo) where T : struct { throw new PlatformNotSupportedException(); }
 
-        /// <summary>
-        /// __m256i _mm256_setzero_si256 (void)
-        ///   HELPER
-        /// __m256 _mm256_setzero_ps (void)
-        ///   HELPER
-        /// __m256d _mm256_setzero_pd (void)
-        ///   HELPER
-        /// </summary>
-        public static Vector256<T> SetZeroVector256<T>() where T : struct { throw new PlatformNotSupportedException(); }
```

Comment:
I recommend that just add new APIs in this PR, and remove these old ones later (after implementing new APIs in JIT). That would make the workflow simpler and convenient to compare the codegen quality.

Reply:
You are suggesting 4 total PRs?

This PR, which adds the new APIs
The CoreFX PR, which adds the new APIs
Another PR, which removes the existing APIs and updates the tests
The CoreFX PR, which removes the existing APIs

Reply:
It shouldn't need to be. I've already validated the implementation locally and any fixup (if something was missed) can happen in the third PR.